test/types/index.test-d.ts

```diff
@@ -1,9 +1,9 @@
-import { expectType } from 'tsd'
+import {expectType} from 'tsd'
```

Comment:
what is the linting here?
```suggestion
import { expectType } from 'tsd'
```